hangman_lunch.rb

```diff
@@ -0,0 +1,161 @@
+#Lila and Corinne
+
+# works through lunch, first req of dinner (not penalizing repeat guesses on same letter)
+
+require "colorize"
+
+class Hangman
+	def initialize
+
+		# library
+		@words = ["orca", "cheese", "cake", "pumpkin", "sun", "apple", "cloud", "paint", "bird", "travel", "lollipop", "ada"]
+
+		@answer_word = choose_word # array of characters
+		@answer_copy = @answer_word.dup
+		@word_display = @answer_copy.fill("_ ")
+		@guessed_letters = [] # guessed letter array
+
+		@default_guesses = 7
+		@wrong_count = 0
+
+		#creates
+		start
+	end
+
+# Display (art, word)
+
+
+	def start
+		puts "Let's play HANGMAN."
+		update_visuals
+		@keep_playing = true
+
+		play
+	end
+
+	def play
+		# checks for loss due to use of all guesses
+		while @keep_playing == true
+			if @wrong_count < 7 && win != true
```

Comment:
Although `win` just seems like a variable here, it is actually a method call. By using the `win` method call here and in line 44 - the code needs to call that method multiple times. I would recommend storing that in a local variable prior to the conditional execution so you don't need to do that.